AssertionError in `TestSWINUNETR`

```

[2024-11-30T19:24:25.796Z] ======================================================================

[2024-11-30T19:24:25.796Z] FAIL: test_filter_swinunetr_0 (tests.test_swin_unetr.TestSWINUNETR)

[2024-11-30T19:24:25.796Z] ----------------------------------------------------------------------

[2024-11-30T19:24:25.796Z] Traceback (most recent call last):

[2024-11-30T19:24:25.796Z]   File "/usr/local/lib/python3.10/dist-packages/parameterized/parameterized.py", line 620, in standalone_func

[2024-11-30T19:24:25.796Z]     return func(*(a + p.args), **p.kwargs, **kw)

[2024-11-30T19:24:25.796Z]   File "/opt/monai/tests/test_swin_unetr.py", line 137, in test_filter_swinunetr

[2024-11-30T19:24:25.796Z]     self.assertTrue(len(loaded) == 157 and len(not_loaded) == 2)

[2024-11-30T19:24:25.796Z] AssertionError: False is not true
```

--- a/monai/networks/nets/swin_unetr.py
+++ b/monai/networks/nets/swin_unetr.py
@@ -75,7 +75,7 @@ class SwinUNETR(nn.Module):
         dropout_path_rate: float = 0.0,
         normalize: bool = True,
         norm_layer: type[LayerNorm] = nn.LayerNorm,
-        patch_norm: bool = True,
+        patch_norm: bool = False,
         use_checkpoint: bool = False,
         spatial_dims: int = 3,
         downsample: str | nn.Module = "merging",
@@ -102,7 +102,7 @@ class SwinUNETR(nn.Module):
             dropout_path_rate: drop path rate.
             normalize: normalize output intermediate features in each stage.
             norm_layer: normalization layer.
-            patch_norm: whether to apply normalization to the patch embedding.
+            patch_norm: whether to apply normalization to the patch embedding. Default is False.
             use_checkpoint: use gradient checkpointing for reduced memory usage.
             spatial_dims: number of spatial dims.
             downsample: module used for downsampling, available options are `"mergingv2"`, `"merging"` and a

